package-installer/src/main/java/com/vmware/pscoe/iac/installer/Installer.java

```diff
@@ -1416,6 +1422,7 @@ private static void readVrliImportProperties(final Input input) {
         userInput(input, Option.VRLI_VROPS_INTEGRATION_USER, "  vROps integration username");
         passInput(input, Option.VRLI_VROPS_INTEGRATION_PASSWORD, "  vROps integration password");
         userInput(input, Option.VRLI_VROPS_INTEGRATION_AUTH_SOURCE, "  vROps integration auth source", "local");
+        userInput(input, Option.VRLI_USE_OLD_ALERTS_API, "  Use old vRLI Alerts API", Boolean.FALSE);
```

Comment:
I don't understand it, not sure whoever uses the installer will. Can we add some description to the output what this will do?

Reply:
Actually there is a documentation how to use it

Reply:
The installer is usually used by clients. Clients that don't really understand anything. These prompts should be able to guide them... I think we should aim for the messages to be easier to understand. 

I would ask my self for example:
- Why would I use an old API, when I can use the new one?